.vscode/settings.json

```diff
@@ -3,5 +3,19 @@
         "tests"
     ],
     "python.testing.unittestEnabled": false,
-    "python.testing.pytestEnabled": true
+    "python.testing.pytestEnabled": true,
+    "workbench.colorCustomizations": {
+        "commandCenter.border": "#e7e7e799",
+        "sideBar.border": "#6204bd",
+        "statusBar.background": "#48038b",
+        "statusBar.foreground": "#e7e7e7",
+        "statusBarItem.hoverBackground": "#6204bd",
+        "statusBarItem.remoteBackground": "#48038b",
+        "statusBarItem.remoteForeground": "#e7e7e7",
+        "titleBar.activeBackground": "#48038b",
+        "titleBar.activeForeground": "#e7e7e7",
+        "titleBar.inactiveBackground": "#48038b99",
+        "titleBar.inactiveForeground": "#e7e7e799"
+    },
+    "peacock.color": "#48038b"
```

Comment:
Should all this be checked in?

Reply:
Nope will remove

Reply:
You'll need to force rm it if you dont want it checked in anymore, since git was already tracking, I think.